README.md

```diff
@@ -16,7 +16,7 @@ Features include:
 provide the user provider for the core [SecurityBundle](https://symfony.com/doc/current/book/security.html).
 
 **Caution:** This bundle is developed in sync with [symfony's repository](https://github.com/symfony/symfony).
```

Comment:
this should just be removed entirely. The message about being developed in sync is a left-over from the Symfony 2.0 vs 2.1 era when BC was not a thing yet